packages/data-stores/src/site/selectors.ts

```diff
@@ -22,7 +22,11 @@ export const isNewSite = ( state: State ) => !! state.newSite.data;
 export const getSite = ( state: State, siteId: number | string ) => {
 	return (
 		state.sites[ siteId ] ||
-		Object.values( state.sites ).find( ( site ) => site && new URL( site.URL ).host === siteId )
+		Object.values( state.sites ).find( ( site ) => site && new URL( site.URL ).host === siteId ) ||
```

Comment:
Can we update the documentation describing this new behavior?

Reply:
@jeyip I did go ahead and quickly add some inline documentation as suggested.